cdflib/epochs.py

```diff
@@ -172,14 +173,15 @@ def _compose_date(years, months, days,
                  '<m8[m]', '<m8[s]', '<m8[ms]', '<m8[us]', '<m8[ns]')
         vals = (years, months, days, hours, minutes, seconds,
                 milliseconds, microseconds, nanoseconds)
+
         return sum(np.asarray(v, dtype=t) for t, v in zip(types, vals)
                    if v is not None)
 
     @classmethod
     def to_datetime(cls, cdf_time: Union[int, Sequence[int]],
                     to_np: bool = False) -> List[datetime.datetime]:
         """
-        Encodes the epoch(s) into Python datetime.  Precision is only
+        Encodes the epoch(s) into Numpy datetime64.  Precision is only
```

Comment:
This may be wrong, but from the casting it appeared to be the case.  Please correct if I overlooked something.